src/main/java/com/github/housepower/jdbc/statement/ClickHousePreparedInsertStatement.java

```diff
@@ -75,49 +63,20 @@ private void addParameters() throws SQLException {
 
     @Override
     public void clearBatch() throws SQLException {
+        this.block.initWriteBuffer();
+        new ValuesWithParametersInputFormat(fullQuery, posOfData).fillBlock(block);
     }
 
     @Override
     public int[] executeBatch() throws SQLException {
```

Comment:
This will make the connection stat to WAIT_INSERT, but what if we want to do select queries?